src/pages/Profile.jsx

```diff
@@ -32,29 +35,45 @@ function Profile() {
     },
   ];
 
+  const [user, setUser] = useState({});
+
+  useEffect(async () => {
+    const res = await AFCBackend.get(`/users/${userId}`, {
+      headers: {
+        'Content-Type': 'application/json',
+      },
```

Comment:
you don't really have to specify this header when you request using afcbackend, let's get rid of it